bccsp/sw/fileks_test.go

```diff
@@ -89,7 +89,7 @@ func TestBigKeyFile(t *testing.T) {
 
 	_, err = ks.GetKey(ski)
 	assert.Error(t, err)
-	expected := fmt.Sprintf("Key with SKI %s not found in %s", hex.EncodeToString(ski), ksPath)
+	expected := fmt.Sprintf("key with SKI %s not found in %s", hex.EncodeToString(ski), ksPath)
```

Comment:
%x can be used instead of the hex package

Reply:
addressed in #936